lib/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from "./discovery.js"
 export * from "./RevHub.js"
+export * from "./open-parent-rev-hub.js"
```

Comment:
Let's just call this `open-rev-hub.ts`, since there will be additional opening functions that should go in the same file (for organization and to facilitate shared non-exported code).

Reply:
The file got auto-renamed when renaming the method. Would it make sense to keep the name for now, and rename the file when we implement the method for opening children?